src/main/java/com/realtime/nyc/subway/scheduler/TripPublishScheduler.java

```diff
@@ -11,7 +11,7 @@
 
     private final SubwayDataPublisherService subwayDataPublisherService;
 
-    @Scheduled(fixedDelay = 30000)
+    @Scheduled(fixedRateString = "${data.fetch.rate.ms}")
```

Comment:
was actually a bit confused between Rate and Delay considering why publish if the last one hasn't completed but I guess this makes more sense.
Did you tried `fixedRate` with value from placeholder or `fixedRateString` will only work?

Reply:
We can asynchronously handle these requests which will omit the cases where the previous requests take longer than expected, but I feel this will guarantee data generation for every 30th second on timestamp.

`fixedRateString` because the properties are read as a string and it will not be possible to cast them to long, since the placeholder value has to be final to be passed to an annotation. [might be wrong here, looking for other possible solutions, anything in mind?]

Reply:
I think this is perfect, no over-complication needed here.
Kindly go ahead and merge with squash message.